.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+# Pull Request Summary
+(Instructions: this, and all subsequent sections of text should be removed and filled in as appropriate.)   
+Please describe the PR summary
+
+## Description
+Provide a detailed description of what this PR does.
+What bug does it fix, or what feature does it add?
+Is a change of answers expected from this PR?
+
+### Issue(s) addressed
+* Is there an issue associated with this development (bug fix, enhancement, new feature)?    
+Please add a reference to a related issue(s) in WW3 repository (Follow [link](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)).
+Link the issues to be closed with this PR, whether in this repository, or in another repository.
+(Remember, issues should always be created before starting work on a PR branch!).  
+Note that properly "linked issues" (either automatic links, or manual ones using the correct keywords) will be automatically closed when the PR is merged.
+
+- fixes #<issue_number>
+- fixes noaa-emc/ww3/issues/<issue_number>
+
+### Check list  
+* Is your feature branch up to date with the authoritative repository (NOAA/develop)?
+* Make sure you have checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop), [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop) and [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number)
+* @mentions of the person or team responsible for reviewing proposed changes.
```

Comment:
Maybe this line should be: 

Reviewers: @mentions of suggested reviewers of the proposed changes

Reply:
Done